app/javascript/widget/views/Home.vue

```diff
@@ -5,7 +5,11 @@
   >
     <spinner size="" />
   </div>
-  <div v-else class="home">
+  <div
+    v-else
+    class="home"
+    @keydown.esc="closeChat"
```

Comment:
I understood the implementation here. This would close the chat only if the focus is on the content area. Shall we remove this as we have a close button below it already?